clouddq/main.py

```diff
@@ -582,18 +582,17 @@ def main(  # noqa: C901
                 )
                 logger.info("Job completed successfully.")
             else:
-                logger.warning(
-                    "'--target_bigquery_summary_table' was not provided. "
-                    "It is needed to append the dq summary results to the "
-                    "provided target bigquery table. This will become a "
-                    "required argument in v1.0.0"
-                )
                 if summary_to_stdout:
                     logger.warning(
                         "'--summary_to_stdout' was set but does"
                         " not take effect unless "
                         "'--target_bigquery_summary_table' is provided"
                     )
```

Comment:
Does this still make sense? Now --target_bigquery_summary_table is a required parameter anyway.

Reply:
yes i agree , I haven't removed it because if we want to give log message, but that doesn't add much value as we are throwing error just after. Removed it now.